packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -317,6 +318,79 @@ curl -v -X POST \
 ##### Response Example
 
 
+```json
+{
+  "id": "00ub0oNGTSWTBKOLGLNR",
+  "status": "ACTIVE",
+  "created": "2013-07-02T21:36:25.344Z",
+  "activated": null,
+  "statusChanged": null,
+  "lastLogin": null,
+  "lastUpdated": "2013-07-02T21:36:25.344Z",
+  "passwordChanged": "2013-07-02T21:36:25.344Z",
+  "profile": {
+    "firstName": "Isaac",
+    "lastName": "Brock",
+    "email": "isaac.brock@example.com",
+    "login": "isaac.brock@example.com",
+    "mobilePhone": "555-415-1337"
+  },
+  "credentials": {
+    "password": {},
+    "provider": {
+      "type": "IMPORT",
+      "name": "IMPORT"
+    }
+  },
+  "_links": {
+    "activate": {
+      "href": "https://${yourOktaDomain}/api/v1/users/00ub0oNGTSWTBKOLGLNR/lifecycle/activate"
+    },
+    "self": {
+      "href": "https://${yourOktaDomain}/api/v1/users/00ub0oNGTSWTBKOLGLNR"
+    }
+  }
+}
+```
+
```

Comment:
The 'Response Example' should be after the 'Request Example', with the order being: 'Create User with Password Import Inline Hook', 'Request Example', 'Response Example'.

Currently, I think this 'Response Example' is overwriting the response example for 'Create User with Imported Hashed Password'.

Reply:
@motikagan-okta 
I think what @rameshbhojan-okta has here is OK.  As I see it, the response object is identical in the import hashed password and import password with hooks case.  So what we have is

1) Import hashed password request (old)
2) Response (appears new, but is actually the same thing we had before)
3) Import password with hook request (new)
4) Response (old, but still appropriate, since the responses are the same for both requests (1) and (3)).

Reply:
Yes, apologies, @rameshbhojan-okta. I read the diff view wrong, @edjohns-okta is right--it's okay as-is.